admin/plugins.php

```diff
@@ -0,0 +1,405 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets plugin info for the given plugin slug from WordPress.org.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $plugin_slug The string identifier for the plugin in questions slug.
+ *
+ * @return array Array of plugin data, or empty if none/error.
+ */
+function perflab_query_plugin_info( string $plugin_slug ) {
+	$plugin = plugins_api(
+		'plugin_information',
+		array(
+			'slug'   => $plugin_slug,
+			'fields' => array(
+				'short_description' => true,
+				'icons'             => true,
+			),
+		)
+	);
+
+	if ( is_wp_error( $plugin ) ) {
+		return array();
+	}
+
+	if ( is_object( $plugin ) ) {
+		$plugin = (array) $plugin;
+	}
+
+	return $plugin;
+}
+
+/**
+ * Returns an array of WPP standalone plugins.
+ *
+ * @since n.e.x.t
+ *
+ * @return array of wpp standalone plugins as slugs.
+ */
+function perflab_get_standalone_plugins() {
+	return array(
+		'webp-uploads',
+		'performant-translations',
+		'dominant-color-images',
+	);
+}
+
+/**
+ * Renders plugin UI for managing standalone plugins within PL Settings screen.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_plugins_ui() {
+	require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+	require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+	$standalone_plugins = array();
+	foreach ( perflab_get_standalone_plugins() as $managed_standalone_plugin_slug ) {
+		$standalone_plugins[ $managed_standalone_plugin_slug ] = array(
+			'plugin_data' => perflab_query_plugin_info( $managed_standalone_plugin_slug ),
+		);
+	}
+
+	if ( empty( $standalone_plugins ) ) {
+		return;
+	}
+	?>
+	<div class="wrap">
+		<h1><?php esc_html_e( 'Performance Plugins', 'performance-lab' ); ?></h1>
+		<p><?php esc_html_e( 'The following standalone performance plugins are available for installation.', 'performance-lab' ); ?></p>
+		<div class="wrap">
+			<form id="plugin-filter" method="post">
+				<div class="wp-list-table widefat plugin-install wpp-standalone-plugins">
+					<h2 class="screen-reader-text"><?php esc_html_e( 'Plugins list', 'default' ); ?></h2>
+					<div id="the-list">
+						<?php
+						foreach ( $standalone_plugins as $standalone_plugin ) {
+							perflab_render_plugin_card( $standalone_plugin['plugin_data'] );
+						}
+						?>
+					</div>
+				</div>
+			</form>
+		</div>
+		<div class="clear"></div>
+	</div>
+	<?php
+}
+
+/**
+ * Renders individual plugin cards.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $plugin_data Plugin data from the WordPress.org API.
+ */
+function perflab_render_plugin_card( array $plugin_data ) {
```

Comment:
Since this function includes code copied from core, please add a `@link` to the GitHub URL 
```suggestion
/**
 * Renders individual plugin cards.
 *
 * This is adapted from `WP_Plugin_Install_List_Table::display_rows()` in core.
 *
 * @since n.e.x.t
 *
 * @see WP_Plugin_Install_List_Table::display_rows()
 * @link https://github.com/WordPress/wordpress-develop/blob/0b8ca16ea3bd9722bd1a38f8ab68901506b1a0e7/src/wp-admin/includes/class-wp-plugin-install-list-table.php#L467-L830
 *
 * @param array $plugin_data Plugin data from the WordPress.org API.
 */
function perflab_render_plugin_card( array $plugin_data ) {
```

This will facilitate keeping this code up to date as the corresponding code in core evolves.